Cargo.toml

```diff
@@ -16,6 +16,10 @@ name = "graphql-schema"
 path = "src/api/schema/gen.rs"
 required-features = ["default-no-api-client"]
 
+[profile.release]
+lto = true
+codegen-units = 1
```

Comment:
My only concern here is that we had `codegen-units = 1` previously and decided to remove it because of how much longer it makes the build take (if I remember correctly).

Reply:
Yes, lto=true+codegen-units=1 require more time, but we do not need this too often.

Reply:
We were already using `codegen-units = 1` previously. The only real change here, I think, is `lto`.